src/contracts/arbitrum/AaveV3ArbWSTETHListingPayload.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+import {AaveV3Arbitrum} from 'aave-address-book/AaveV3Arbitrum.sol';
+import {IGenericV3ListingEngine, AaveV3ListingArbitrum} from 'aave-helpers/v3-listing-engine/AaveV3ListingArbitrum.sol';
+
+/**
+ * @title List wstETH on AaveV3Arbitrum
+ * @author Llama
+ * @dev This proposal lists wstETH on Aave V3 Arbitrum
+ * Governance: https://governance.aave.com/t/arc-add-support-for-wsteth-on-arbitrum-aave-v3/11387/11
+ * Snapshot: https://snapshot.org/#/aave.eth/proposal/0x402647b83c436aecbe2fe404870f08767b5509225cbe606913e50801f87f5db8
+ */
+contract AaveV3ArbWSTETHListingPayload is AaveV3ListingArbitrum {
+  address public constant WSTETH = 0x5979D7b546E38E414F7E9822514be443A4800529;
+  address public constant WST_ETH_PRICE_FEED = 0x230E0321Cf38F09e247e50Afc7801EA2351fe56F;
```

Comment:
I couldn't find this price feed on the Chainlink Documentation. Where did you get it from?

Reply:
Shared via DM.